test/langtools/tools/javac/lambda/lambdaExpression/LambdaTest6.java

```diff
@@ -26,11 +26,9 @@
  * @bug 8003280
  * @summary Add lambda tests
  *   Test bridge methods for certain SAM conversions
- *   Tests that jdk.internal.lambda.disableEagerInitialization=true creates a
- *   get$Lambda method for non-capturing lambdas
+ *   Test the set of generated methods
  * @compile LambdaTest6.java
  * @run main LambdaTest6
- * @run main/othervm -Djdk.internal.lambda.disableEagerInitialization=true LambdaTest6
```

Comment:
This line was added by JDK-8232806 (https://hg.openjdk.java.net/jdk/jdk/rev/27c2d2a4b695).
I assume you want to move the test case for JDK-8232806 to test/jdk/java/lang/invoke?  If so,
BridgeMethod.java should be looked at too.

Reply:
I have removed all the `disableEagerInitialization` tests from `BridgeMethod.java`. It is now restored to its pre-JDK-8232806 state.